css/parser/src/parser/at_rule.rs

```diff
@@ -27,7 +27,7 @@ where
             }
         };
 
-        match &*name.to_lowercase() {
+        match &*name.to_ascii_lowercase() {
```

Comment:
My rust knowledge is still very young (if you known good books, I will be glad your recommendations), but `to_ascii_lowercase` is valid, because at-rules can be written using non ascii characters

Reply:
Yes, you are right.